app/src/main/java/org/mifos/selfserviceapp/ui/fragments/TransferProcessFragment.java

```diff
@@ -127,8 +128,16 @@ public void startTransfer() {
      */
     @OnClick(R.id.btn_cancel_transfer)
     public void cancelTransfer() {
-        getActivity().getSupportFragmentManager().popBackStack();
-        getActivity().getSupportFragmentManager().popBackStack();
+        Snackbar snackbar = Snackbar
```

Comment:
@luckyman000786 We already have Snack bar utils class, please use this.
https://github.com/openMF/self-service-app/blob/development/app/src/main/java/org/mifos/selfserviceapp/utils/Toaster.java

Reply:
@therajanmaurya I tried with Snack bar utils class but failed because I was unable to set a OnClickListener